src/pages/Index.vue

```diff
@@ -2,18 +2,115 @@
   <Layout>
 
     <!-- Learn how to use images here: https://gridsome.org/docs/images -->
-    <h1>Welcome to the Galaxy Mentoring Network</h1>
+    <g-image alt="Example image" src="~/favicon.png" width="135" />
+    <header class="header">
+        <b-container class=".b-container">
+        <b-row>
+          <b-col lg="6" md="6" sm="12">
+        <h1>Mentorship...for you!</h1>
 
-    <p>Mentorship...for you!</p>
+        <p>An all in one program targeted to speed up your growth!</p>
+        <b-btn class=".btn">Apply Now</b-btn>
+            </b-col>
+            <b-col>
+              <img src="./assets/images/illustration.png" img-alt="two people sitting in front of a computer" width="400px">
+          </b-col>
+          </b-row>
+        </b-container>
+    </header>
+    <b-container class=".b-container">
+     <b-card-group deck>
+         <b-card class=".b-card"
+            title="Community"
+           img-src="./assets/images/gcc2019-grouppicture.jpg"
+           img-alt="Community members at Galaxy Community Conference, 2019"
+           img-top
+           border-radius="10px"
+           >
+            <b-card-text>
+               GMN is birthed from the Galaxy Community and has her at the center of affairs.
+             </b-card-text>
+          </b-card>
+          <b-card class=".b-card"
+              title="Collaboration"
+             img-src="./assets/images/linkedin-sales-solutions-QgYvORVDdd8-unsplash.jpg"
+             img-alt="Four hands joined together"
+             img-top
+             border-radius="10px"       
+              >
+             <b-card-text>
+                  The program is founded and sustained by the pillars of collaboration.
+              </b-card-text>
+           </b-card>
+           <b-card class=".b-card"
+                title="Growth"
+                img-src="./assets/images/krakenimages-Y5bvRlcCx8k-unsplash.jpg"
+                img-alt="Corporate lady smiling"
+                img-top
+                border-radius="10px"       
+                >
+                <b-card-text>
+                    Your growth and expectations are our focus, and your growth drives us.
+                 </b-card-text>
+           </b-card>
+     </b-card-group>
+    </b-container>
+     <b-container class=".b-container">
+       <b-row style="background-color:#e9ecef;" class="p-4">
```

Comment:
Please put the style in the class

Reply:
This gave me an issue...I put the style in the class and the padding (`p-4`) didn't have an effect on the row anymore

Reply:
Please how do I do it correctly :( ?

Reply:
You can define a specific class, e.g. `index-row-bg-color`